EVSVesuvio/scripts/handle_config.py

```diff
@@ -64,8 +65,14 @@ def setup_config_dir(config_dir):
 
 def setup_expr_dir(cache_dir, experiment):
     expr_path = os.path.join(cache_dir, "experiments", experiment)
-    __mk_dir('experiment', expr_path)
-    copyfile(os.path.join(VESUVIO_PACKAGE_PATH, "config", VESUVIO_INPUTS_FILE), input_file_path(cache_dir, experiment))
+    if not os.path.isdir(expr_path):
+        __mk_dir('experiment', expr_path)
+        copyfile(os.path.join(VESUVIO_PACKAGE_PATH, "config", VESUVIO_INPUTS_FILE), input_file_path(cache_dir, experiment))
+
+
+def setup_default_ipfile_dir():
+    if not os.path.isdir(VESUVIO_IPFOLDER_PATH):
+        copytree(os.path.join(VESUVIO_PACKAGE_PATH, "config", "ip_files"), VESUVIO_IPFOLDER_PATH)
```

Comment:
This copies over the `__pycache__` directory and the `__init_.py` file, can we exclude those?